plugin/core/edit.py

```diff
@@ -24,7 +25,11 @@ def parse_workspace_edit(workspace_edit: WorkspaceEdit) -> WorkspaceChanges:
             uri = text_document['uri']
             version = text_document.get('version')
             edits = document_change.get('edits')
-            changes.setdefault(uri, ([], version))[0].extend(edits)
+            changes.setdefault(uri, ([], version))[0].extend(
+                # currently LSP code supprots only `TextEdit`s
+                # TODO: Extend WorkspaceEdit to support `AnnotatedTextEdit` and `SnippetTextEdit`
+                [edit for edit in edits if is_text_edit(edit)]
+            )
```

Comment:
Not sure if it's this part but this PR seems to break something related to applying code edits.

If you try to apply fixes in this file by saving it with LSP-eslint and `"source.fixAll.eslint": true,` https://github.com/rchl/test-admin-js/blob/main/src/admin/options.ts then the syntax somewhat breaks

Reply:
Yes, I can verify that it breaks the syntax.